src/app/pages/Map/Map.tsx

```diff
@@ -299,66 +303,108 @@ export default function MapBox(): JSX.Element {
   return (
     <>
       <InputPage hidden={showMapPage}>
+        <AppName>MidWay</AppName>
         <InputContainer>
-          <h1>See U There</h1>
-
-          <InputPageButton onClick={() => onSet()}>
-            set location
-          </InputPageButton>
-          <LocationInput id="locationInput"></LocationInput>
+          <GeocoderBox>
+            <LocationInput id="locationInput"></LocationInput>
+            <InputPageButton onClick={() => onSet()}>
+              set location
+            </InputPageButton>
+            <ClearAllButton onClick={() => onClear()}>
+              <DeletX />
+            </ClearAllButton>
+          </GeocoderBox>
           <YourLocationInput
             locationName1={locationName1}
             locationName2={locationName2}
           />
-          <InputPageButton onClick={() => onClear()}>
-            clear all locations
-          </InputPageButton>
-          <NavigationButton onClick={() => showMap()}>
-            <NavigationButtonMapIcon />
-          </NavigationButton>
+          <NavigationContainerMap>
+            <NavigationButton onClick={() => showMap()}>
+              <NavigationButtonMapIcon />
+            </NavigationButton>
+          </NavigationContainerMap>
         </InputContainer>
       </InputPage>
       <MapPage hidden={showMapPage}>
-        <span>Drivingdistance: {distance} m</span>
-
         <MapContainer ref={mapContainer} className="map-container" />
-        <NavigationButton onClick={() => showMap()}>
-          <NavigationButtonBackIcon />
-        </NavigationButton>
-        <NavigationButton onClick={switchToMore}>
-          <NavigationButtonMoreIcon />
-        </NavigationButton>
+        <NavigationContainerInput>
+          <NavigationButton onClick={() => showMap()}>
+            <NavigationButtonInputIcon />
```

Comment:
Would it make sense to give the Icon a more generic name, so that you can use it for the future on other pages? Maybe just "PlusIcon" so that it's clear for the reader how that Icon looks like.